cesql/spec.md

```diff
@@ -212,7 +216,8 @@ runtime errors.
 
 ### 3.4. Operators
 
-The following tables show the operators that MUST be supported by a CESQL evaluator.
+The following tables show the operators that MUST be supported by a CESQL evaluator. When evaluating an operator,
+a CESQL engine MUST attempt to cast the operands to the correct types. If this type casting fails, a _Cast_ error will be returned, along with the zero value for the return type of the expression.
```

Comment:
do we define the zero value for each type? While probably obvious we should say it's: zero, "" and `false`.

Reply:
We do now :)